spec/features/reminders_spec.rb

```diff
@@ -47,21 +48,21 @@
           click_on "Continue"
 
           # - Which subject did you do your postgraduate initial teacher training (ITT) in?
-          choose I18n.t("additional_payments.forms.eligible_itt_subject.answers.#{args[:subject]}")
+          choose "Yes"
```

Comment:
On master we're eligible for both policies, but the test set up indicates we only care about ECP.
This test loops twice, once with itt_academic_year of "2018 to 2019" and claim year of 2022, and once with itt_academic_year of "2019 to 2020" and claim year of 2023.  Given a claim year of 2022 and itt year of 2018 the displayed subjects should only be `mathematics` for ECP policies, same for 2023 claim year and 2019 itt year. What's happening on master is the subjects for the LUP policy are also being returned so there are multiple options available. I'm not sure if this was intended as we don't set the LUP policy up to be eligible. Choosing yes doesn't change what we care about in this test (that a reminder is created) so wen't with that option rather than setting up the answers to be LUP eligible too.